src/modules/bids-asks/actions/cancel-order.js

```diff
@@ -18,10 +17,7 @@ export const cancelOrder = (orderID, marketID, outcome, orderTypeLabel, callback
     dispatch(updateOrderStatus(orderID, CLOSE_DIALOG_CLOSING, marketID, outcome, orderTypeLabel))
     augur.api.CancelOrder.cancelOrder({
       meta: loginAccount.meta,
-      _market: marketID,
       _orderId: orderID,
-      _outcome: outcome,
-      _type: orderTypeLabel === BUY ? 1 : 2,
```

Comment:
I know this is deleted code, but just a heads-up that "buy" and "sell" values are 0 and 1 instead of 1 and 2 (this changed a while back on the trading contracts).